overcloud.yml

```diff
@@ -27,6 +27,19 @@
           src: "firstboot-nvme.yaml.j2"
           dest: "/home/stack/firstboot-nvme.yaml"
         when: nvme is defined
+      - block:
+          - name: adjust max number of concurrent builds to default (override infrared)
+            ini_file:
+                path: /var/lib/config-data/puppet-generated/nova/etc/nova/nova.conf
+                section: DEFAULT
+                option: max_concurrent_builds
+                value: 10
+            become: yes
+
+          - name: restart nova services
+            command: "systemctl restart tripleo_nova_*"
```

Comment:
use "systemctl restart openstack-nova-*"

Reply:
On my 16.1 Undercloud the services for nova are tripleo_nova_*

```
[stack@undercloud ~]$ sudo systemctl list-units | grep nova
  tripleo_nova_api.service                                                                                                             loaded active running   nova_api container                                                                                                          
  tripleo_nova_api_cron.service                                                                                                        loaded active running   nova_api_cron container                                                                                                     
  tripleo_nova_compute.service                                                                                                         loaded active running   nova_compute container                                                                                                      
  tripleo_nova_conductor.service                                                                                                       loaded active running   nova_conductor container                                                                                                    
  tripleo_nova_scheduler.service                                                                                                       loaded active running   nova_scheduler container                                                                                                    
  tripleo_nova_api_cron_healthcheck.timer                                                                                              loaded active waiting   nova_api_cron container healthcheck                                                                                         
  tripleo_nova_api_healthcheck.timer                                                                                                   loaded active waiting   nova_api container healthcheck                                                                                              
  tripleo_nova_compute_healthcheck.timer                                                                                               loaded active waiting   nova_compute container healthcheck                                                                                          
  tripleo_nova_conductor_healthcheck.timer                                                                                             loaded active waiting   nova_conductor container healthcheck                                                                                        
  tripleo_nova_scheduler_healthcheck.timer                                                                                             loaded active waiting   nova_scheduler container healthcheck                                                                                        
```

Reply:
The updated logic should work for both cases.